app/views/submit_confirmation.scala.html

```diff
@@ -16,20 +16,20 @@
 
 @import config.AppConfig
 
-@import repositories.declaration.Submission
+@import models.Declaration
 
-@(submission: Submission)(implicit req: Request[_], messages: Messages, cfg: AppConfig)
+@(submission: Declaration)(implicit req: Request[_], messages: Messages, cfg: AppConfig)
```

Comment:
Change variable name to `declaration`
```suggestion
@(declaration: Declaration)(implicit req: Request[_], messages: Messages, cfg: AppConfig)
```